eventual/src/App.tsx

```diff
@@ -1,14 +1,18 @@
 import React from 'react';
 import './App.css';
+import Heading from './heading/heading'
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <h1>Eventual</h1>
-      </header>
-    </div>
-  );
+class App extends React.Component {
```

Comment:
Why did you change this to a class component? I'd prefer we keep using functional components if possible. 🙏

Reply:
Was just testing things out, thanks for letting me know. Committed changes here https://github.com/bldg14/eventual/pull/9/commits/33f2624db35a16ff9d899f5970ed0114ee113acf